homeworks/misha.kolosov_mwkolosov/homework_7/main.js

```diff
@@ -0,0 +1,17 @@
+function fib(maxNum) {
+    const arr = [0, 1];
+    let current = 0;
+
+    while (current <= maxNum) {
+        current += arr[arr.length - 2] || 1;
```

Comment:
`arr[arr.length - 2] || 1` - wrong check for a valid value... if element equals `0` it will return `1`